schemas/date.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+
+  "type":"string",
+  "pattern":"^\d{4}-((0[1-9])|(1[012]))-((0[1-9]|[12]\d)|3[01])$"
```

Comment:
I also don't think we need the capture groups. They would just be necessary for parsing the String via regexp, but we just want to validate.